doc/fundamentals/distributed_computing.md

```diff
@@ -30,41 +30,68 @@ If you use the `CONSTANT` strategy, Pachyderm will start the number of workers t
 
 If you use the `COEFFICIENT` strategy, Pachyderm will start a number of workers that is a multiple of your Kubernetes cluster’s size. To use this strategy, set the field `coefficient` to a double. For example, if your Kubernetes cluster has 10 nodes, and you set `"coefficient": 0.5`, Pachyderm will start five workers. If you set it to 2.0, Pachyderm will start 20 workers (two per Kubernetes node).
 
-**NOTE:** The parallelism_spec is optional and will default to `“coefficient": 1”`, which means that it'll spawn one worker per Kubernetes node for this pipeline if left unset. 
+**NOTE:** The parallelism_spec is optional and will default to `“coefficient": 1`, which means that it'll spawn one worker per Kubernetes node for this pipeline if left unset. 
 
 ## Spreading Data Across Workers (Glob Patterns)
 
 Defining how your data is spread out among workers is arguably the most important aspect of distributed computation and is the fundamental idea around concepts like Map/Reduce. 
 
 Instead of confining users to just data-distribution patterns such as Map (split everything as much as possible) and Reduce (_all_ the data must be grouped together), Pachyderm uses [Glob Patterns](https://en.wikipedia.org/wiki/Glob_(programming)) to offer incredible flexibility in defining your data distribution. 
 
- Glob patterns are defined by the user for each input of a pipeline and tell Pachyderm how to divide the input data into individual "datums" that can be processed independently. 
+ Glob patterns are defined by the user for each `atom` within the `input` of a pipeline, and they tell Pachyderm how to divide the input data into individual "datums" that can be processed independently. 
 
 ```
-"inputs": [
-    {
-      "repo": {
-        "name": "string"
-      },
-      "glob": "string"
+"input": {
+    "atom": {
```

Comment:
This seems somewhat confusing.  Can you somehow make it more explicit that you can't actually have more than one input types at the top level?